src/js/decorators/track-password-manager-usage.js

```diff
@@ -0,0 +1,48 @@
+module.exports = function trackPasswordManagerUsage() {
+  var fireGAEvent = require( 'helpers/fire-ga-event' );
+  var passwordField = document.getElementById( 'field-password' );
+  var form = document.getElementById( 'nlx-form' );
+  var start = 0;
+  var end = 0;
+  var time_spent_typing;
+
+  passwordField.addEventListener( 'keypress', watchKeyTiming );
+  form.addEventListener( 'submit', function() {
+    if ( form.getAttribute( 'lock-state' ) === 'ldap' ) {
+      report();
+    }
+  });
+
+  function watchKeyTiming(event) {
+    if ( start === 0 && event.target.value.length > 0 ) {
+      start = Date.now();
+    }
+
+    end = Date.now();
+    console.log( 'Handler for keypress called: ' + event.which );
```

Comment:
Now that we know it works, maybe we can remove this console.log?

Reply:
Yes, we should. Also, because it leaks the user's password if they type it by hand, one character at a time into the console log